doc/README.md

```diff
@@ -38,7 +38,7 @@ Building
 The following are developer notes on how to build Bitcoin Core on your native platform. They are not complete guides, but include notes on the necessary libraries, compile flags, etc.
 
 - [Dependencies](dependencies.md)
-- [macOS Build Notes](build-osx.md)
+- [macOS Build Notes](build-macos.md)
```

Comment:
Correct me, but last name change was a fix because osx=os10. So next change would be when apple decide to merge ios and macos since current layout does not really have such verbal limitations.

Reply:
It would be easier to skip this one, thats for sure. But what if its gonna stick for another decade :)